druid/helper.lua

```diff
@@ -5,6 +5,7 @@ local const = require("druid.const")
 
 local M = {}
 
+local system_name = sys.get_sys_info().system_name
```

Comment:
Constants should be named with UPPER_CASE. Also you can move it to `const.lua` flle with two constants: SYS_INFO and CURRENT_SYSTEM_NAME